.github/ISSUE_TEMPLATE/help.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Help
+about: 'Need help setting up a VSMP?'
+title: ''
+labels: help
```

Comment:
Could we use the __question__ label instead of __help__? Question already exists and will avoid confusion with the __help wanted__ label.

Reply:
Sounds great! see 43a26ef